Casks/font-open-huninn.rb

```diff
@@ -1,11 +1,12 @@
 cask 'font-open-huninn' do
   version '1.0'
-  sha256 '20b18c056608ec078d22f1677b52fd1b616def1512f17be5d37831f09f517d5d'
+  sha256 '8b0d0e8fd396a763371ae21bcf7140926e7837d5851d108108b858de42c9b5e2'
+
+  # codeload.github.com/justfont/open-huninn-font was verified as official when first introduced to the cask
+  url 'https://codeload.github.com/justfont/open-huninn-font/zip/master'
 
-  # justfont.com/huninn was verified as the official when first introduced to the cask
-  url 'https://raw.githubusercontent.com/justfont/open-huninn-font/master/font/jf-openhuninn-1.0.ttf'
   name 'jf-openhuninn-1.0'
   homepage 'https://justfont.com/huninn/'
 
-  font 'jf-openhuninn-1.0.ttf'
+  font 'open-huninn-font-master/font/jf-openhuninn-1.0.ttf'
```

Comment:
Interpolate the version.

```suggestion
  font "open-huninn-font-master/font/jf-openhuninn-#{version}.ttf"
```